test/parallel/test-repl-.editor.js

```diff
@@ -31,6 +31,17 @@ function run({input, output, event}) {
   stream.emit('data', input);
   replServer.write('', event);
   replServer.close();
+
+  if (!checkTerminalCodes) {
+    while (found.includes(terminalCode))
+      found = found.replace(terminalCode, '');
```

Comment:
@princejwesley Yeah, I’ve updated it to use a regex… I’m not particularly a fan of duplicating strings that are hard to scan, but if you prefer a plain regex literal over what I’ve just pushed, I can go with that too